csharp/Adapter/documentation/Mobius_API_Documentation.md

```diff
@@ -142,13 +142,6 @@
             Used for collect operation on RDD
             
         
-###<font color="#68228B">Microsoft.Spark.CSharp.Core.IRDDCollector</font>
```

Comment:
Since you are not changing public interface, you could exclude this file and the XML file from your PR.